From 6bb46921bdfb345a2b15aed596d96c57c1ff694b Mon Sep 17 00:00:00 2001
From: James Kosin <jkosin@apache.org>
Date: Thu, 10 Nov 2011 02:34:55 +0000
Subject: [PATCH] OPENNLP-361: force JAVA_CMD to only contain short path
 names... Thanks to Aliaksandr Autayeu for his testing and pushing

git-svn-id: https://svn.apache.org/repos/asf/incubator/opennlp/trunk@1200104 13f79535-47bb-0310-9956-ffa450edef68
---
 opennlp-distr/src/main/bin/opennlp.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opennlp-distr/src/main/bin/opennlp.bat b/opennlp-distr/src/main/bin/opennlp.bat
index cdaad0448..3c42051a1 100644
--- a/opennlp-distr/src/main/bin/opennlp.bat
+++ b/opennlp-distr/src/main/bin/opennlp.bat
@@ -25,7 +25,7 @@ IF "%JAVA_CMD%" == "" (
 	IF "%JAVA_HOME%" == "" (
 		SET JAVA_CMD=java 
 	) ELSE (
-		SET JAVA_CMD="%JAVA_HOME%\bin\java"
+		FOR %%A IN ("%JAVA_HOME%") DO SET JAVA_CMD=%%~sfA\bin\java
 	)
 )
 
